ludwig/features/image_feature.py

```diff
@@ -125,7 +156,7 @@ def add_feature_data(
                         img = resize_image(
                             img,
                             (im_height, im_width),
-                            feature['resize_method'],
+                            feature['preprocessing']['resize_method'],
```

Comment:
Shouldn't this be `preprocessing_parameters['resize_method']`?

Reply:
updated